frontend/src/App.tsx

```diff
@@ -1,18 +1,24 @@
 import React, {useState} from 'react';
 import './App.css';
 import axios from "axios";
+import ProductList from "./supplier/ProductList";
+import useProducts from "./supplier/useProducts";
 
-function App() {
 
-  const [message, setMessage] =useState();
+function App() {
+    const productHook = useProducts();
+    const [message, setMessage] = useState();
 
-  axios.get("/api/supplier")
-      .then((response => response.data))
-      .then(setMessage)
+    axios.get("/supplier/products")
```

Comment:
du hast deine Axios-Aufrufe doch unten in den useProducts ausgelagert, wieso rufst du sie hier nochmal auf?